src/form/types/types.ts

```diff
@@ -790,7 +790,7 @@ export type IDeclarationCertificate = {
   corrector?: Partial<{ type: RelationForCertificateCorrection }>
   hasShowedVerifiedDocument?: boolean
   payments?: Payment
-  data?: string
+  certTemplateId?: string
```

Comment:
Can we rename this to `certificateTemplateId` please? Better be explicit with naming

Reply:
sure